### PR TITLE
Feature/234  fix gift entry merge conflict

### PR DESCRIPTION
**Pull request by psadie-sfdo**
# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# Features Intended for Future Release

# Features for Elevate Customers

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: force-app/main/default/classes/Gifts.cls [hunk 1]
```diff
@@ -146,7 +180,8 @@ public with sharing class Gifts {
     private void setElevateBatchIds(List<DataImport__c> dataImports) {
         elevateBatchIds = new Set<String>();
         for (DataImport__c dataImport : dataImportsById.values()) {
-            if (dataImport?.Payment_Status__c == ElevateTransactionStatus.AUTHORIZED.name()) {
+            if (dataImport?.Status__c == BDI_DataImport_API.bdiDryRunValidated
```
Comment (voduyemi) [inline force-app/main/default/classes/Gifts.cls:183]: Non-blocking: Can we extract this guard clause into a method of its own so we have an understandable reference point always?

**Review by voduyemi — COMMENTED** [review]
Looks good to me @psadie-sfdo. Just had that one non-blocking comment.

**Review by voduyemi — APPROVED** [review]
Thanks for making the update! Looks good.

**Review by bdvorachek — APPROVED** [review]

**Comment by bdvorachek** [comment]
The original conflict PR, #6610 , should close once this PR is merged.

**Pull request merged**